run_cumulus_task.py

```diff
@@ -5,6 +5,7 @@
 import os
 import sys
 
+from cumulus_logger import CumulusLogger
```

Comment:
Is this how others tasks should use this logger too?

Reply:
yes